RawEventProcessor.cpp

```diff
@@ -366,6 +395,29 @@ bool RawEventProcessor::process_syscall_event(const Event& event) {
                         _pid = atoi(f.RawValuePtr());
                         _builder->SetEventPid(_pid);
                     }
+                    if (fname == SV_PPID) {
+                        _ppid = atoi(f.RawValuePtr());
+                    }
+                    add_field = true;
```

Comment:
No as this would change the current functionality.  In the original code these case statements didn't exist and these clauses would instead have hit the default, which is 'add_field = true'. Therefore, if the clauses hit the new case statements then they are no longer hitting the default, and the action of the default has to be included in these case statements, regardless of whether any other actions are taken or not. Hence, the 'add_field = true' statements must reside outside of the ifs.